BicycleParkingStation/examples/example-normalized.jsonld

```diff
@@ -15,7 +15,7 @@
         "type": "Relationship",
         "object": {
             "type": "Geometry",
-            "Geometry.wkt": {
+            "Geometry.asWkt": {
```

Comment:
ok to the change, but Why is it?

Reply:
Because the JSON-LD context uses this: https://github.com/smart-data-models/dataModel.OSLO/blob/master/notes_context.jsonld#L181